cherab/tools/observers/group/base.py

```diff
@@ -32,22 +32,22 @@ class Observer0DGroup(Node):
     all observers, or each observer can be assigned with individual value.
 
     :ivar list names: A list of observer names.
-    :ivar list pipelines: A list of all pipelines connected to each observer in the group.
     :ivar list/RenderEngine render_engine: Rendering engine used by the observers.
                                            Note that if the engine is shared, changing its
                                            parameters for one observer in a group will affect
                                            all observers.
-    :ivar list/float min_wavelength: Lower wavelength bound for sampled spectral range.
+    :ivar list/int spectral_bins: The number of smaller sub-spectrum rays the full spectrum will be divided into.
+    :ivar list/int spectral_rays: The number of smaller sub-spectrum rays the full spectrum will be divided into.
```

Comment:
Thank you! Everything is fine except for this copy-paste error in the docstring.

Reply:
It should be fixed now.